src/EFCore.PG.Trgm/EFCore.PG.Trgm.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+
+    <AssemblyName>Npgsql.EntityFrameworkCore.PostgreSQL.Trgm</AssemblyName>
+    <RootNamespace>Npgsql.EntityFrameworkCore.PostgreSQL.Trgm</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.0.0" />
+  </ItemGroup>
+
+</Project>
```

Comment:
Please align this file with other plugin projects and don't forget to add you as an author.

@roji Should you be mentioned as an author? While the EF provider is your work, the plugin is made by @ins0mniaque who contributed it to us.

Reply:
No, I don't need to be in the author list, since I haven't done anything :)

Reply:
593c77fc6768cf1f32e4a0c04a7fdf6ff0806aed fixes this.

Reply:
LGTM